modules/core/src/test/java/org/apache/ignite/internal/processors/cache/distributed/GridExchangeFreeSwitchTest.java

```diff
@@ -159,10 +161,27 @@ public void testBaselineNodeLeftOnFullyRebalancedCluster() throws Exception {
         }
     }
 
+    /**
+     * Checks Partition Exchange is regular in case of fixed baseline, when IGNITE_PME_FREE_SWITCH_DISABLED is true.
+     */
+    @Test
+    public void testBaselineNodeLeftOnFullyRebalancedClusterPmeFreeDisabled() throws Exception {
```

Comment:
Use @WithSystemProperty on test instead of try-finally

Reply:
done